### ISSUE TITLE
Changing CUBEJS_PRE_AGGREGATIONS_SCHEMA causes problems

### ISSUE THREAD

**Issue by JeffreyPalmer** [cube-js/cube#1779]
**Describe the bug**
When `CUBEJS_PRE_AGGREGATIONS_SCHEMA` was introduced the default value of `stb_pre_aggregations` was no longer used anywhere. When I booted up my environment (not realizing that this had been changed), I had a cascading set of errors relating to missing schemas, invalid REDIS entries, etc.

**To Reproduce**
1. Run cube server locally with REDIS and pre-aggregations enabled
2. Change the value of `CUBEJS_PRE_AGGREGATIONS_SCHEMA` to something new
3. Restart the server
4. See redis-related errors in the log

(Unfortunately I can't seem to replicate this at this point, but I am logging it as requested in in slack.)

**Expected behavior**
No errors are raised

**Screenshots**

**Version:**
`0.25.6`

**Additional context**
Happy to discuss this further if necessary. Thanks!

**Comment by ovr** [comment]
Thank you for bug reporting.

When we changed a default value for `CUBEJS_PRE_AGGREGATIONS_SCHEMA` in https://github.com/cube-js/cube.js/commit/e5bdf3dfbd28d5e1c1e775c554c275304a0941f3#diff-6271a8d4f777316c8ac5e440f0ada73ce1ff88123115e4509e35b4aae2aac888R162 from `stb_pre_aggregations ` to `devServer ? 'dev_pre_aggregations' : 'prod_pre_aggregations'`, We didn't think that it can be a BC for any setups.

Probably, The problem comes from Redis (we miss `schema` from prefix). I started to work on it, I will check codebase to find any possible places where We can miss prefix (CUBEJS_PRE_AGGREGATIONS_SCHEMA) inside keys.
